src/StreamJsonRpc.Tests/JsonRpcClient20InteropTests.cs

```diff
@@ -352,4 +494,29 @@ public async Task ServerReturnsOurRequestIdAsString()
         string result = await invokeTask;
         Assert.Equal("pass", result);
     }
+
+    private async Task<WeakReference> InvokeWithProgressParameter_NoMemoryLeakConfirm_Helper()
```

Comment:
>InvokeWithProgressParameter_NoMemoryLeakConfirm_Helper [](start = 38, length = 54)

This method needs a `[MethodImpl(MethodImplOptions.NoInlining)]` attribute. Otherwise the JIT can inline the method and defeat the reason we separated it to begin with, causing the test to sometimes fail making it look like a leak where there actually is none.